### PR TITLE
Patch 4

### PR DESCRIPTION
**Pull request by JackRabbit1312**

### CODE REVIEW

**Comment by cemathey** [comment]
You need to change your RCON game server password because you've posted it on github @JackRabbit1312

**Comment by cemathey** [comment]
Also could you make your branch up to date with `master` on the main repo, resolve any merge conflicts on your end and give us a description of what you're changing and why?

The Discord is linked on the main repo you can also swing by and chat with us there.

File: .env [hunk 1]
```diff
@@ -93,13 +93,13 @@ RCONWEB_EXTERNAL_ADDRESS=localhost
...
-HLL_HOST=
+HLL_HOST=176.57.168.240
 
 # The HLL RCON port (this is not the game or query port)
-HLL_PORT=
+HLL_PORT= 28416
 
 # The RCON  password of your game server
-HLL_PASSWORD=
+HLL_PASSWORD=8910
```
Comment (MarechJ) [inline .env:102]: Dude you just leaked your password.

Change it immediately @JackRabbit1312

**Comment by MarechJ** [comment]
> Also could you make your branch up to date with `master` on the main repo, resolve any merge conflicts on your end and give us a description of what you're changing and why?
> 
> The Discord is linked on the main repo you can also swing by and chat with us there.

Most like someone with a custom fork trying to merge back the updates.
It's quite disappointing to see the amount of custom stuff built or inspired from that project but barely any coming back to this repo :(

Closing this

**Pull request closed without merging**